sunkit_pyvista/plotter.py

```diff
@@ -304,9 +310,10 @@ def plot_solar_axis(self, length=2.5, arrow_kwargs={}, **kwargs):
                               direction=(0, 0, length),
                               scale='auto',
                               **defaults)
-        color = kwargs.get('color', np.nan)
-        arrow_mesh.add_field_array([color], 'color')
-        self.plotter.add_mesh(arrow_mesh, **kwargs)
+        color = kwargs.pop('color', 'white')
```

Comment:
Could you create a new method called `_extract_color` that:

- Takes `kwargs` as an input
- Returns a color as an RGB tuple?

This was we can reduce code duplication in several of the new lines you've added.

Reply:
Yes, this makes more sense I shall do that

Reply:
Yes I'll add thsi in.